appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1400,6 +1400,15 @@ private static int upgradeWebProperties(Map<String, JSONValue> componentProperti
       // No properties need to be modified to upgrade to version 4.
       srcCompVersion = 4;
     }
+	if (srcCompVersion < 5) {
```

Comment:
watch out for indentation here; should be 2 spaces for a new line, and 4 if it is a continuation of the previous line.